src/main/java/rlbot/manager/BotManager.java

```diff
@@ -98,8 +100,7 @@ private void doLoop() {
             }
 
             try {
-                // Fetch the latest game tick packet at 60 Hz.
-                Thread.sleep(16);
+                Thread.sleep(1000 / refreshRate.get());
```

Comment:
If we sleep exactly the time between frames it will eventually skip a frame because of the compute time outside the sleep state.

Reply:
Temporary fix in e66c9ea

Reply:
Might cause duplicate frames now though